modules/minimap2/1.0/config/default.yaml

```diff
@@ -0,0 +1,36 @@
+lcr-modules:
+    
+    minimap2:
+
+        # TODO: Update the list of available wildcards, if applicable
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_fastq: "__UPDATE__"
+            reference_build: "__UPDATE__"
+
+        scratch_subdirectories: []
+
+        options:
+            minimap2: '-ax map-ont -L --MD -Y -R "@RG\tID:{sample_id}\tLB:{sample_id}\tPL:ONT\tSM:{sample_id}"'
+            samtools: '-bhS'
+
+        conda_envs:
+            minimap2: "{MODSDIR}/envs/minimap2-2.24.yaml"
+            samtools: "{MODSDIR}/envs/samtools-1.9.yaml"
+            
+        threads:
+            minimap2: 4
```

Comment:
Just wonder how long this tool takes to run and if we should consider giving it more resources here?

Reply:
I gave it more resources (10) and now it takes ~24 hours

Reply:
Kostia's point is fair - it makes sense to set the default here MUCH higher. I think you should make 24 the default.